src/test/java/io/nats/client/ConnectTests.java

```diff
@@ -256,21 +231,20 @@ public void testConnectNoRandomize() throws IOException, InterruptedException {
                     Options options = new Options.Builder().noRandomize().servers(servers).build();
                     Connection nc = Nats.connect(options);
                     try {
-                        assertTrue(Connection.Status.CONNECTED == nc.getStatus(), "Connected Status");
+                        assertConnected(nc);
 
                         if (nc.getConnectedUrl().equals(ts1.getURI())) {
                             one++;
                         } else {
                             two++;
                         }
                     } finally {
-                        nc.close();
-                        assertTrue(Connection.Status.CLOSED == nc.getStatus(), "Closed Status");
+                        closeConnectionAssertClosed(nc);
                     }
                 }
 
-                assertTrue(one == 10, "always got one");
-                assertTrue(two == 0, "never got two");
+                assertEquals(one, 10, "always got one");
+                assertEquals(two, 0, "never got two");

```

Comment:
nitty for sure, but again how the assertions were designed.

Reply:
I see linter complaints about these... thanks.